tests/activityDetection.test.js

```diff
@@ -3,10 +3,10 @@ const childProcess = require('child_process');
 
 let activityDetection;
 
-describe('Tests for Activity Detection', function() {
+describe('Tests for Activity Detection', function () {
 
     var originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
```

Comment:
Is there no way to do this dynamically? What if we run this on a low configuration system?